communicationPlugIn-Kafka/src/main/java/com/cgi/hexagon/commplugin/kafka/OrderJson.java

```diff
@@ -0,0 +1,67 @@
+package com.cgi.hexagon.commplugin.kafka;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class OrderJson {
+    String shopOrderId;
+    String goodsId;
+    int quantity;
+
+    JsonNode metadata;
```

Comment:
add placementDate property as well please. Based on our last discussion this is not a parameter that we want to set on realization, the shop is going to send this as well